interview-questions.md

```diff
@@ -8,36 +8,66 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort and a model called Student. The cohort model has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: The name of the foreign key is cohort_id followed by a integer. You can fix the mistake by adding it to the migration file. The key would be in the cohort model. 
 
-Researched answer:
+Researched answer: You can run the command rake db:rollback, then go to migration and edit foreign key.
 
 2. Which RESTful routes must always be passed params? Why?
 
-Your answer:
+Your answer: Show, Update, Create, Destroy. Because these routes target a specific instance in the database, and id will target the correct instance.
 
-Researched answer:
+Researched answer: The Create Route requires parameters to pass the data needed to create the resource. Update Route, requires parameters to pass the data that needs to be updated. Delete Route requires parameters to indicate the specific resource that should be deleted.
 
 3. Name three rails generator commands. What is created by each?
 
-Your answer:
+Your answer: Rails db:create, will create the database for the rails app. Rails s, will open and run the server, and Rails db:migrate, will migrates files made to the corresponding route. 
```

Comment:
This question is asking about generator commands, not any rails commands. For example, rails generate model is a generator command that is used to create a model.